store/postgres/org.go

```diff
@@ -87,6 +105,14 @@ func (s Store) CreateOrg(ctx context.Context, orgToCreate model.Organization) (m
 func (s Store) ListOrg(ctx context.Context) ([]model.Organization, error) {
 	var fetchedOrgs []Organization
 	err := s.DB.WithTimeout(ctx, func(ctx context.Context) error {
+		nr := newrelic.DatastoreSegment{
```

Comment:
We can create a function that only accept `ctx`, `Collection` and `Operation` args and return `nr`.
So everywhere in PG, calling it would be
```
nr := createNRSegment(ctx, "organizations", "List Organizations")
defer nr.End()
```